src/summary/documentation/vitals-card.test.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { mockPatient } from "../../../__mocks__/patient.mock";
+import { cleanup, render, wait } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import VitalsCard from "./vitals-card.component";
+import { of } from "rxjs/internal/observable/of";
+import * as openmrsApi from "./vitals-card.resource";
+
+describe("<VitalsCard/>", () => {
+  let patient: fhir.Patient, match;
+  const mockVitalsResponse = [
+    {
+      id: 1463379216000,
+      date: "2016-05-16T06:13:36.000+00:00",
+      systolic: 161,
+      diastolic: 72,
+      pulse: 22,
+      temperature: 37,
+      oxygenation: 30
+    },
+    {
+      id: 144048423500012,
+      date: "2015-08-25T06:30:35.000+00:00",
+      systolic: 156,
+      diastolic: 64,
+      pulse: 173,
+      temperature: 37,
+      oxygenation: 41
+    }
+  ];
+  afterEach(() => {
```

Comment:
This is incorrect usage of `@testing-library/react`'s cleanup function. The cleanup function has to actually be **invoked**.

```suggestion
  afterEach(cleanup)
```

Or (equivalently):

```suggestion
  afterEach(() => {
    cleanup()
  })
```

See related https://testing-library.com/docs/react-testing-library/api#cleanup